client/termsetting/handlers/numeric.discrete.ts

```diff
@@ -74,16 +74,17 @@ async function showBinsMenu(self, div: any) {
 	div.append('div').style('padding', '10px').style('text-align', 'center').html('Getting distribution data ...<br/>')
 	try {
 		if (!self.vocabApi) throw `Missing .vocabApi{} [numeric.discrete showBinsMenu()]`
-
+		const tw = await fillTermWrapper({ term: self.term, q: self.q }, self.vocabApi) //the q for the discrete mode may need to be initialized
```

Comment:
should instead do `{ term: self.term, q: {mode:'continuous'}}`? so that backend always gets continuous mode

Reply:
also i don't understand why it should consider discrete mode? discrete shouldn't be used for violin request